ethcore/src/evm/evm.rs

```diff
@@ -66,6 +67,8 @@ pub enum Error {
 	MutableCallInStaticContext,
 	/// Likely to cause consensus issues.
 	Internal(String),
+	/// Wasm runtime error
+	Wasm(String),
```

Comment:
As long as it's strictly for human-readable messages then that's fine. If we ever finding ourself matching on it we can refactor to make it an error.